bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -272,12 +274,12 @@ public synchronized void login() {
                 if (!loginCozyTouch()) {
                     return;
                 }
-            } else {
-                loginTahoma();
             }
 
             if (thingConfig.isDevMode()) {
                 initializeLocalMode();
+            } else {
+                loginTahoma();
```

Comment:
With this change, I believe this will be called also for Cozytouch. Please check again.

Reply:
You are right I have to check the Cloud portal setting. I managed to solve it differently, since the developer mode make sense only with Tahoma portal

Reply:
Looks better